backend/Recodo.BLL/Services/MailService.cs

```diff
@@ -0,0 +1,28 @@
+using MimeKit;
+using MimeKit.Text;
+
+namespace Recodo.BLL.Services
+{
+    public sealed class MailService
```

Comment:
Can you reuse an existing mail service and add this method there

Reply:
Ok, i`ll made one service out of two

Reply:
Not fixed
We still have two services and yours doesn't use SendGrid